cmd/qri.go

```diff
@@ -128,6 +129,11 @@ func (o *QriOptions) Init() (err error) {
 	}
 	setNoPrompt(o.NoPrompt)
 
+	repoErr := lib.QriRepoExists(o.repoPath)
```

Comment:
I added this check in the `Init` function here because it seems like all our Qri CLI commands either call this directly in their respective `Complete` function (like [here](https://github.com/qri-io/qri/pull/1540/files#diff-8ff128e26e876899f9196f8b77b3e358R73)) or many call it indirectly via other methods like `DatasetMethods` [here](https://github.com/qri-io/qri/blob/master/cmd/init.go#L72).

Is it a fair assumption that `Init` will be called before every command is run, though? I assume yes since initializing instance state seems like a pre-req to running any command but want to be sure. Is there also any function that we always run before each cmd call? I'd like to put something like this there instead of assuming `Init` will be called in one way or another (and skip the logic if the command is `setup`) but I couldn't track one down other than something like `NewQriCommand` which feels too soon to check for a config since the user wouldn't have a chance to run `qri setup` yet

Reply:
This is exactly the right spot. I don't think it's explicitly stated, but it's certainly the intention that `Init` will setup state before any command is run, except for `setup`. Could you modify the comment describing `Init` to state as much, and then switch this from Draft to Ready?